src/app/youtube/services/youtube.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { YoutubeService } from './youtube.service';
+
+describe('YoutubeService', () => {
+  let service: YoutubeService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(YoutubeService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
```

Comment:
Please review all your tests if they do same thing like here(they do nothing) remove it, or create correct test.

Reply:
I removed it. Tests were added automatically